src/Site_WP_Docker.php

```diff
@@ -142,6 +142,28 @@ public function generate_docker_compose_yml( array $filters = [] ) {
 		];
 		$mailhog['networks']     = $network_default;
 
+		// postfix configuration.
```

Comment:
There is repeatability in the array appending as well. To reduce this as well and make it re-usable for packages, you can possibly create a new `site-docker-compose-utils`. Create a function for postfix and assign `EE\SiteUtils\get_postfix_array()` and append it to the main array going for mustaching.

Similar to postfix, you can do this for the common services like nginx, mailhog, php, db etc and create the utils to get there configs in an array so people can use them as defaults easily. This change would be quite hepful here as well as in current `site-command`.